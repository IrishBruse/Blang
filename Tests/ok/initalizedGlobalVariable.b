variable 3;

main() {
    extrn printf;
    printf("variable %d\n", variable);
}
