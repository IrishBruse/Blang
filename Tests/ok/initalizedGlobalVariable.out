variable 3350647
